tests/test_builder.py

```diff
@@ -16,7 +16,7 @@
 import unittest
 from unittest import TestCase
 
-import testing_utils
+import tests.testing_utils as testing_utils
```

Comment:
Good catch. We should use the future absolute_import everywhere quite likely. As part of porting to Python 3 sometimes.